### ISSUE TITLE
Add relevant NYC polygons

### ISSUE THREAD

**Issue by chriswhong** [BetaNYC/citygram-nyc#4]
Give the user the ability to choose from Community Districts, Precincts, etc. to receive updates in instead of just a radius.  <<Recommended by Richard Robbins on the BetaNYC email list.

**Comment by dirkkelly** [comment]
@excepttheweasel, you expressed some interest in this story tonight. Here it is for your reference.

**Comment by tribzi** [comment]
I agree that I would be very helpful to choose to receive data by community board, council districts or even by borough. This adds some appeal to legislators as well who can then select their particular jurisdiction.

**Comment by davidem00** [comment]
you might also consider the ability to describe collections of points as well (with radii, possibly zero).  This would allow for the designation of a subway line, or a set of non-adjacent properties, etc.
